code/modules/shuttle/navigation_computer.dm

```diff
@@ -10,10 +10,11 @@
 	var/list/jumpto_ports = list() //hashset of ports to jump to and ignore for collision purposes
 	var/obj/docking_port/stationary/my_port //the custom docking port placed by this console
 	var/obj/docking_port/mobile/shuttle_port //the mobile docking port of the connected shuttle
+	var/list/locked_traits = list(ZTRAIT_RESERVED, ZTRAIT_CENTCOM, ZTRAIT_AWAY, ZTRAIT_REEBE) //traits forbided for custom docking
```

Comment:
Just being picky, but it's forbidden, not forbided

Reply:
Honestly I think a list of whitelisted traits would be better tbh. Thoughts @qwertyquerty ?